Update Firefox user agent string (#34050)
mdn/content: files/en-us/web/http/headers/user-agent/firefox/index.md

| 1.0 (WebView Tablet)             | Mozilla/5.0 (Linux; Android 7.0) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Focus/1.0 Chrome/59.0.3029.83 Safari/537.36        |
| 6.0 (GeckoView)                  | Mozilla/5.0 (Android 7.0; Mobile; rv:62.0) Gecko/62.0 Firefox/62.0                                                                     |

> The Klar user agent is the same as Focus.

Reviewer: ```suggestion
The Klar user agent is the same as [Focus](#focus_for_ios).
```

> The Klar user agent is the same as [Focus](#focus_for_ios).

Reviewer: ```suggestion
The Klar user agent is the same as [Focus](/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox).
```
this is the only edit i made, so testing to see the CI issues.

> The Klar user agent is the same as [Focus](/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox).

Reviewer: ```suggestion
The Klar user agent is the same as [Focus](/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox#focus_for_ios).
```
